Week3/homework/step3-bonus.js

```diff
@@ -3,8 +3,8 @@
 const values = ['a', 'b', 'c', 'd', 'a', 'e', 'f', 'c'];
 
 function makeUnique(arr) {
-  // Replace this comment and the next line with your code
-  console.log(arr);
+  const unique = arr.filter((x, y, z) => z.indexOf(x) === y);
+  return unique;
```

Comment:
See my comment below with respect to creating the variable `unique`.

Reply:
Also this one is shorthened.